File: client/src/app/_models/photo-filter-dto.ts
import { SortOrder } from '../_enums/sort-order-enum';



export class PhotoFilterDto{
    constructor(page = 0){
        this.Page = page;
        this.Tags = [];
        this.Countries = [];
        this.Cities = [];
        this.Years = [];
        this.Order = SortOrder.Random;
        this.AlreadyShownPhotos = [];
    }
    Page: number;
    Tags: number[];
    Countries: number[];
    Cities: number[];
    Years: number[];
    Order: SortOrder;
    AlreadyShownPhotos: number[]
}

File: backend/AuthTest.Core/DTO/Photo/PhotoFilterDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AuthTest.Core.DTO.Photo
{
    public class PhotoFilterDto
    {
        public int Page { get; set; }
        public List<int> Tags { get; set; }
        public List<int> Countries { get; set; }
        public List<int> Cities { get; set; }
        public List<int> Years { get; set; }
    }
}
